### ISSUE TITLE
list(Enum) is invalid

### ISSUE THREAD

**Issue by iansan5653** [microsoft/pyright#285]
**Describe the bug**
Per the [spec](https://docs.python.org/3/library/enum.html), instances of `enum.Enum` are iterable. Therefore, they should be valid to pass to the `list` constructor, but Pyright does not allow it.

**To Reproduce**
Simply pass any enum to `list()`.

**Expected behavior**
A list should be generated of all the values of the enum.

**Screenshots or Code**
```py
from enum import Enum, auto

class Sport(Enum):
  FOOTBALL = auto()
  BASKETBALL = auto()
  SOCCER = auto()
  TENNIS = auto()

all_sports = list(Sport);
```
Result:
> No overloads for 'list(Sport)' match parameters
> Argument types: (Type[Sport])

**VS Code extension or command-line**
I'm running VSCode extension version 1.0.71.

**Comment by erictraut** [comment]
Thanks for the bug report. I've repro'ed the bug, and I'll working on a fix.

**Comment by erictraut** [comment]
This is now fixed in 1.0.72, which I just published. Thanks again for the bug report!
